companies/4wave.json

```diff
@@ -6,7 +6,7 @@
     "categories": [
         "ads"
     ],
-    "name": "blueleads GmbH",
+    "name": "  blueleads GmbH      ",
```

Comment:
🚫 **[recordlint]** <[pass-schema](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record did not pass schema validation: “The value at /name is an invalid string.”

Raw error:
```json
{
    "path": "$.name",
    "pointer": "/name",
    "message": "The value at /name is an invalid string.",
    "data": "  blueleads GmbH      "
}
```